src/main/groovy/io/saagie/plugin/dataops/clients/SaagieClient.groovy

```diff
@@ -377,19 +379,19 @@ class SaagieClient {
         Request projectsUpdateJobRequest = saagieUtils.getProjectUpdateJobFromDataRequestFromParams(job)
         returnData = updateProjectJobWithGraphQLFromRequest(projectsUpdateJobRequest)
 
-        addJobVersionFromConfiguration()
+        addJobVersionFromConfiguration(job, jobVersion)
 
         return returnData
     }
 
     String updateProjectJobFromParams(job, jobVersion) {
         logger.info('Starting updateProjectJob task')
-        logger.debug('Using config [job={}, jobVersion={}]', configuration.job, configuration.jobVersion)
+        logger.debug('Using config [job={}, jobVersion={}]', job, configuration.jobVersion)
```

Comment:
why logging configuration.jobVersion when this method takes jobVersion in param?